src/playground-project.ts

```diff
@@ -712,6 +712,57 @@ const typeEnumToMimeType = (type?: string) => {
       return 'text/html; charset=utf-8';
     case 'css':
       return 'text/css; charset=utf-8';
+    // taken from MDN's common MIME types
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+    case 'svg':
```

Comment:
Just in the interest of brevity, I think I'd probably skip these:

```
bmp
tif
tiff
avi
mp4
mpeg
ogv
3gp
3g2
aac
oga
opus
wav
mid
midi
```

Because we have better alternatives (png/webp for images, weba for audio, webm for video). We can always tell users to use one of those, or to just explicitly specify the mime type in the config file if they really want (using the `contentType` field).